imap_processing/tests/ultra/unit/test_ultra_l1b.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+import pytest
+import xarray as xr
+
+from imap_processing.ultra.l1b.ultra_l1b import create_dataset, ultra_l1b
+
+
+@pytest.fixture()
+def mock_data_dict():
+    epoch = np.array([760591786368000000, 760591787368000000, 760591788368000000])
+    data_dict = {"imap_ultra_l1a_45de": xr.Dataset(coords={"epoch": epoch})}
+    return data_dict
+
+
+def test_create_dataset(mock_data_dict):
+    """Tests that dataset is created as expected."""
+    dataset = create_dataset(mock_data_dict, "imap_ultra_l1b_45annotated-de")
+
+    assert "epoch" in dataset.coords
+    assert dataset.coords["epoch"].dtype == "datetime64[ns]"
+    assert dataset.attrs["Logical_source"] == "imap_ultra_l1b_45annotated-events"
+    assert "x_front" in dataset.data_vars
+    assert dataset["x_front"].attrs["UNITS"] == "mm"
+    assert (dataset["x_front"].values == 0).all()
```

Comment:
This will test the shape as well.
```suggestion
    np.testing.assert_array_equal(dataset["x_front"], np.zeros(3))
```